applications/ConstitutiveLawsApplication/tests/cpp_tests/test_thermal_constitutive_laws.cpp

```diff
@@ -0,0 +1,387 @@
+// KRATOS ___                _   _ _         _   _             __                       _
```

Comment:
There is a a lot of copy&paste code in this test. I suggest using moving this to an auxiliary function (you can place this new one in an unnamed namespace).